packages/liquidator/mocha-test/liquidationStrategy.js

```diff
@@ -50,6 +51,62 @@ describe("LiquidatorStrategy", () => {
     result = strat.utils.withdrawProgressPercent(1000, 2000, 1500);
     assert.equal(result.toFixed(2), "50.00");
   });
+  it("hasWithdrawRequestPending", () => {
```

Comment:
I added unit tests for each helper function in the `utility` export for liquidatorStrategy.js